src/components/LoginForm/LoginForm.tsx

```diff
@@ -0,0 +1,72 @@
+import { useState, FormEvent } from 'react';
+import './LoginForm.css';
+import Button from '../Button/Button';
+import { useNavigate } from 'react-router-dom';
+
+function LoginForm() {
+	const [email, setEmail] = useState('');
+	const [password, setPassword] = useState('');
+	const navigate = useNavigate();
+
+	async function handleLogin(event: FormEvent) {
+		event.preventDefault();
+		const response = await fetch('http://localhost:3000/login', {
+			method: 'POST',
+			headers: {
+				'Content-Type': 'application/json',
+			},
+			body: JSON.stringify({ email, password }),
+		});
+
+		console.log(email, password);
```

Comment:
Should probably remove this now you know it works

Reply:
cheers, removed.